package.json

```diff
@@ -52,22 +52,31 @@
   "devDependencies": {
     "axios": "^0.9.1",
     "babel": "^6.5.2",
+    "babel-core": "^6.9.0",
     "babel-eslint": "^6.0.2",
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
     "chai": "^3.5.0",
     "chai-http": "^2.0.1",
+    "enzyme": "^2.3.0",
     "eslint": "^2.8.0",
     "eslint-plugin-babel": "^3.2.0",
     "eslint-plugin-react": "^4.3.0",
+    "expect": "^1.20.1",
```

Comment:
Nope! That was leftover from messing around with an expect library used in an enzyme tutorial.